src/makie.jl

```diff
@@ -851,7 +885,7 @@ function mesh_volume_contour()
     x = y = z = 1:10
     vals = randn(10, 10, 10)
     fig = Figure(resolution=(1200, 400))
-    axs = [Axis3(fig[1, i]; aspect=(1, 1, 1), perspectiveness=0.5) for i = 1:3]
+    axs = [Axis3(fig[1, i]; aspect=(1,1,1), perspectiveness=0.5) for i=1:3]
```

Comment:
Here the space is removed in `(1, 1, 1)` in contrast to the previous review comment. Consistency aids readability. I don't think it's important enough to manually change these things around, but maybe something to keep in mind